docs/articles/clustering/cluster-sharded-daemon-process.md

```diff
@@ -32,5 +32,20 @@ when starting up:
 
 ## Scalability  
 
-This cluster tool is intended for small numbers of consumers and will not scale well to a large set. In large clusters
-it is recommended to limit the nodes the sharded daemon process will run on using a role.
+This cluster tool is intended for small numbers of consumers and will not scale well to a large set. In large clusters it is recommended to limit the nodes the sharded daemon process will run on using a role.
+
+## Push-Based Communication
```

Comment:
Documented the two new pieces of functionality introduced here, which I'll review further down in the code itself